oauth-function-template/CHANGELOG.md

```diff
@@ -16,4 +16,8 @@ Release 1.7.0
 
 Release 1.7.1
 - Changes
-  - Fixed setting up a token when removing credentials
\ No newline at end of file
+  - Fixed setting up a token when removing credentials
+  
+Release 1.7.2
+- Changes
+    - Fixed yargs-parser vulnerability
```

Comment:
I don't think this is the best changelog message. I think that: "Upgrade `convict` to 6.0.0" would be better as external users have no idea about yargs-parser in this context

Reply:
Done.